test/qa/test_dns.py

```diff
@@ -85,10 +87,44 @@ def test_custom_dns_connect(tech, proto, obfuscated, nameserver):
     assert dns.is_unset()
 
 
+@pytest.mark.parametrize("nameserver", lib.DNS)
+@pytest.mark.parametrize("tech,proto,obfuscated", lib.TECHNOLOGIES)
+@pytest.mark.flaky(reruns=2, reruns_delay=90)
+@timeout_decorator.timeout(40)
+def test_custom_dns_off_connect(tech, proto, obfuscated, nameserver):
+    lib.set_technology_and_protocol(tech, proto, obfuscated)
+
+    sh.nordvpn.set.dns(nameserver)
+    assert dns.dns_visible_in_settings(nameserver)
+    assert dns.is_unset()
+
+    sh.nordvpn.set.dns("off")
+    assert dns.dns_visible_in_settings("disabled")
+    assert dns.is_unset()
+
+    output = sh.nordvpn.connect()
```

Comment:
Is there a specific reason why printing the output of connect is necessary? Should just do `assert lib.is_connect_successful(sh.nordvpn.connect())` I think. Although I'm not even sure what purpose this `is_connect_successful` serves, I think if connect fails an exception will happen anyway.